src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -40,9 +39,18 @@ uiModules.get('apps/management')
   this.patternErrors = [];
   this.fetchFieldsError = $translate.instant('KIBANA-LOADING');
 
+  const TIME_FILTER_FIELD_OPTIONS = {
+    NO_DATE_FIELD_SELECTED: {
+      name: $translate.instant('KIBANA-NO_DATE_FIELD_SELECTED')
+    },
+    NO_DATE_FIELDS_IN_INDEX: {
+      name: $translate.instant('KIBANA-NO_DATE_FIELDS_IN_INDEX')
```

Comment:
Really minor suggestion: technically, we're referring to "indices", and not a single index, right? It all depends on how many indices match the index pattern. Maybe it's just me, but when I first started using Kibana, it took awhile for me to understand the difference between index pattern and index. I think renaming these to be plural will help reinforce the relationship, e.g. `NO_DATE_FIELDS_IN_INDICES` and `KIBANA-NO_DATE_FIELDS_IN_INDICES`.